eq-author-api/src/validation/schemas/entities.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$id": "http://example.com/schemas/entities.json",
+  "definitions": {
+    "answer": {
```

Comment:
We don't need to define anything but page title I would have thought. I probably did more in the spike just as I was playing around with it

Reply:
I've removed answer schema def from this PR.